### PR TITLE
Fix some page in rocksmq could not get acked timestamp

### PR DESCRIPTION
**Pull request by aoiasd**
related: https://github.com/milvus-io/milvus/issues/19574 https://github.com/milvus-io/milvus/issues/18288
Signed-off-by: aoiasd <zhicheng.yue@zilliz.com>

### CODE REVIEW

**Comment by aoiasd** [comment]
rerun ut

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl_test.go [hunk 1]
```diff
@@ -184,6 +184,60 @@ func TestRocksmq_Basic(t *testing.T) {
 	assert.Equal(t, string(cMsgs[1].Payload), "c_message")
 }
 
+func TestRocksmq_MutilConsumer(t *testing.T) {
```
Comment (xiaofan-luan) [inline internal/mq/mqimpl/rocksmq/server/rocksmq_impl_test.go:187]: Multi

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go [hunk 2]
```diff
@@ -914,19 +959,19 @@ func (rmq *rocksmq) updateAckedInfo(topicName, groupName string, ids []UniqueID)
...
-		}
 
 		// find min id of all consumer
-		var minBeginID int64 = math.MaxInt64
+		var minBeginID UniqueID = lastID
 		for _, consumer := range consumers {
-			if consumer.beginID < minBeginID {
-				minBeginID = consumer.beginID
+			if consumer.GroupName != groupName {
+				key := constructCurrentID(consumer.Topic, consumer.GroupName)
```
Comment (xiaofan-luan) [inline internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go:967]: remove the CurrentIDSuffix in constructCurrentID

**Pull request merged**